pkg/executor/merge_join_test.go

```diff
@@ -286,13 +286,15 @@ func TestVectorizedShuffleMergeJoin(t *testing.T) {
 			`Shuffle 4150.01 root  execution info: concurrency:4, data sources:[TableReader TableReader]`,
 			fmt.Sprintf(`└─MergeJoin 4150.01 root  inner join, left key:test.%s.a, right key:test.%s.a`, t1, t2),
 			fmt.Sprintf(`  ├─Sort(Build) 3320.01 root  test.%s.a`, t2),
-			`  │ └─TableReader 3320.01 root  data:Selection`,
-			fmt.Sprintf(`  │   └─Selection 3320.01 cop[tikv]  lt(test.%s.b, 5), not(isnull(test.%s.a))`, t2, t2),
-			fmt.Sprintf(`  │     └─TableFullScan 10000.00 cop[tikv] table:%s keep order:false, stats:pseudo`, t2),
+			`  │ └─ShuffleReceiver 3320.01 root  `,
```

Comment:
FYI: This is added since the executor was now built as a part of explain, see change in `buildExplain()`